mmv1/third_party/terraform/services/bigtable/resource_bigtable_table_test.go

```diff
@@ -418,6 +489,35 @@ func testAccBigtableChangeStreamDisabled(t *testing.T) resource.TestCheckFunc {
 	}
 }
 
+func testAccBigtableAutomatedBackupsDisabled(t *testing.T) resource.TestCheckFunc {
```

Comment:
test or Test? Seems inconsistent?

Reply:
Thanks. I've renamed this to `verifyBigtableAutomatedBackupsDisabled` to reduce confusion. This is not a test per-se, but a verification function used as part of the `TestAccBigtableTable_automated_backups_enable` test